server/app/services/export/QuestionJsonSampler.java

```diff
@@ -0,0 +1,525 @@
+      CfJsonDocumentContext jsonApplication = new CfJsonDocumentContext();
+
+      exportEntriesToJsonApplication(jsonApplication, entries);
+
+      return jsonApplication;
+    }
+  }
+
+  class CurrencyJsonSampler implements QuestionJsonSampler<CurrencyQuestion> {
+
+    private final QuestionJsonPresenter currencyJsonPresenter;
+
+    @Inject
+    CurrencyJsonSampler(QuestionJsonPresenter.Factory questionJsonPresenterFactory) {
+      this.currencyJsonPresenter = questionJsonPresenterFactory.create(QuestionType.CURRENCY);
+    }
+
+    @Override
+    public CurrencyQuestion getQuestion(ApplicantQuestion applicantQuestion) {
+      return applicantQuestion.createCurrencyQuestion();
+    }
+
+    @Override
+    public void addSampleData(ApplicantData applicantData, CurrencyQuestion question) {
+      applicantData.putCurrencyDollars(question.getCurrencyPath(), "123.45");
+    }
+
+    @Override
+    public QuestionJsonPresenter getJsonPresenter() {
+      return currencyJsonPresenter;
+    }
+  }
+
+  class DateJsonSampler implements QuestionJsonSampler<DateQuestion> {
+    private final QuestionJsonPresenter dateJsonPresenter;
+
+    @Inject
+    DateJsonSampler(QuestionJsonPresenter.Factory questionJsonPresenterFactory) {
+      this.dateJsonPresenter = questionJsonPresenterFactory.create(QuestionType.DATE);
+    }
+
+    @Override
+    public DateQuestion getQuestion(ApplicantQuestion applicantQuestion) {
+      return applicantQuestion.createDateQuestion();
+    }
+
+    @Override
+    public void addSampleData(ApplicantData applicantData, DateQuestion question) {
+      applicantData.putDate(question.getDatePath(), "2023-01-02");
+    }
+
+    @Override
+    public QuestionJsonPresenter getJsonPresenter() {
+      return dateJsonPresenter;
+    }
+  }
+
+  class EmailJsonSampler extends ContextualizedScalarsJsonSampler<EmailQuestion> {
+    private final QuestionJsonPresenter emailJsonPresenter;
+
+    @Inject
+    EmailJsonSampler(QuestionJsonPresenter.Factory questionJsonPresenterFactory) {
+      this.emailJsonPresenter = questionJsonPresenterFactory.create(QuestionType.EMAIL);
+    }
+
+    @Override
+    public EmailQuestion getQuestion(ApplicantQuestion applicantQuestion) {
+      return applicantQuestion.createEmailQuestion();
+    }
+
+    @Override
+    public void addSampleData(ApplicantData applicantData, EmailQuestion question) {
+      applicantData.putString(question.getEmailPath(), "homer.simpson@springfield.gov");
+    }
+
+    @Override
+    public QuestionJsonPresenter getJsonPresenter() {
+      return emailJsonPresenter;
+    }
+  }
+
+  class EmptyJsonSampler implements QuestionJsonSampler<Question> {
+
+    @Override
+    public CfJsonDocumentContext getSampleJson(QuestionDefinition questionDefinition) {
+      return new CfJsonDocumentContext();
+    }
+
+    @Override
+    public Question getQuestion(ApplicantQuestion applicantQuestion) {
+      return applicantQuestion.createCurrencyQuestion();
```

Comment:
Should this be the currency question?

Reply:
Not quite, we don't want to couple "empty" with static content question, since other question types (currently, enumerator too) should be able to use it.

But returning currency question was a mistake, updated it to return null. Note this is not a problem, since the `getSampleJson` method uses `getQuestion` to pass to it to `addSampleData`, whose implementation is a no-op (below) for `EmptyJsonSampler`.